src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/service/ReferralService.kt

```diff
@@ -124,6 +124,10 @@ class ReferralService(val repository: ReferralRepository) {
       referral.maximumRarDays = it
     }
 
+    if (!update.desiredOutcomeIds.isNullOrEmpty()) {
```

Comment:
this looks like it might make it impossible to explicitly set no desired outcomes. is that the correct behaviour?

Reply:
Changed to ignore if null